src/Microsoft.DotNet.Build.CloudTestTasks/UploadClient.cs

```diff
@@ -0,0 +1,196 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="UploadClient.cs" company="">
+//   
+// </copyright>
+// <summary>
+//   The upload client.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    #region
+
+    using System;
+    using System.Collections;
+    using System.Globalization;
+    using System.IO;
+    using System.Net;
+    using System.Text;
+    using Microsoft.Build.Utilities;
+
+    #endregion
+
+    /// <summary>
+    /// The upload client.
+    /// </summary>
+    public class UploadClient
+    {
+        private TaskLoggingHelper log;
```

Comment:
Naming: private fields are `_underscoreCamelCased`, so `_log`.  This mostly obviates the need for `this.`.